winpr/libwinpr/utils/collections/PubSub.c

```diff
@@ -185,7 +185,9 @@ int PubSub_OnEvent(wPubSub* pubSub, const char* EventName, void* context, const
 	size_t index;
 	wEventType* event;
 	int status = -1;
-	WINPR_ASSERT(pubSub);
+
+	if (!pubSub)
```

Comment:
Do you have a case in mind where the assert was not appropriate (it's public API so yeah we shall not assert, but just wondering) ?